R/template.R

```diff
@@ -1,110 +1,184 @@
 #' needed to calculate the metric.
 #'
 #' @param data The data frame with `truth` and `estimate` columns passed
 #' in from the data frame version of your metric function that called
-#' `metric_summarizer()`.
+#' `numeric_metric_summarizer()`, `class_metric_summarizer()`, or
+#' `prob_metric_summarizer()`
 #'
 #' @param truth The unquoted column name corresponding to the `truth` column.
 #'
 #' @param estimate Generally, the unquoted column name corresponding to
 #' the `estimate` column. For metrics that take multiple columns through `...`
 #' like class probability metrics, this is a result of [dots_to_estimate()].
 #'
-#' @param estimator For numeric metrics, this is left as `NULL` so averaging
-#' is not passed on to the metric function implementation. For classification
-#' metrics, this can either be `NULL` for the default auto-selection of
-#' averaging (`"binary"` or `"macro"`), or a single character to pass along
-#' to the metric implementation describing the kind of averaging to use.
+#' @inheritParams rlang::args_dots_empty
+#'
+#' @param estimator This can either be `NULL` for the default auto-selection of
+#' averaging (`"binary"` or `"macro"`), or a single character to pass along to
+#' the metric implementation describing the kind of averaging to use.
 #'
 #' @param na_rm A `logical` value indicating whether `NA` values should be
 #' stripped before the computation proceeds. The removal is executed in
 #' `metric_vec_template()`.
 #'
-#' @param event_level For numeric metrics, this is left as `NULL` to prevent
-#' it from being passed on to the metric function implementation. For
-#' classification metrics, this can either be `NULL` to use the default
-#' `event_level` value of the `metric_fn` or a single string of either
-#' `"first"` or `"second"` to pass along describing which level should be
-#' considered the "event".
+#' @param event_level This can either be `NULL` to use the default `event_level`
+#' value of the `fn` or a single string of either `"first"` or `"second"`
+#' to pass along describing which level should be considered the `"event"`.
 #'
 #' @param case_weights For metrics supporting case weights, an unquoted
 #' column name corresponding to case weights can be passed here. If not `NULL`,
-#' the case weights will be passed on to `metric_fn` as the named argument
+#' the case weights will be passed on to `fn` as the named argument
 #' `case_weights`.
 #'
-#' @param ... Currently not used. Metric specific options are passed in
-#' through `metric_fn_options`.
 #'
-#' @param metric_fn_options A named list of metric specific options. These
+#' @param fn_options A named list of metric specific options. These
 #' are spliced into the metric function call using `!!!` from `rlang`. The
 #' default results in nothing being spliced into the call.
 #'
-#' @seealso [metric_vec_template()] [finalize_estimator()] [dots_to_estimate()]
-#'
+#' @keywords internal
 #' @export
```

Comment:
Up to you, but when I have a family of functions documented together I always do:

```
#' The docs
#'
#' @name metric-summarizers
NULL

#' @rdname metric-summarizers
#' @export
class_metric_summarizer <- function() {

#' @rdname metric-summarizers
#' @export
numeric_metric_summarizer <- function() {
```

Feels better to give them an overarching `@name` that I can tag each function with, rather than it being tied to `class_metric_summarizer()`

Reply:
done in https://github.com/tidymodels/yardstick/pull/322/commits/c5571802700963715ed46a4f32d59a3086dd6bcf